README.md

```diff
@@ -48,11 +49,13 @@ To deploy workloads with SR-IOV VF this plugin needs to work together with the f
 
   - Retrieves allocated network device information of a Pod
 
-- SR-IOV CNI
-
-  - During Pod creation, plumbs allocated SR-IOV VF to a Pods network namespace using VF information given by the meta plugin
-
-  - On Pod deletion, reset and release the VF from the Pod
+- A CNI capable of consuming the network device allocated to the Pod
```

Comment:
Maybe remove the `SR-IOV VF` in line 46?
or modify it to `To deploy workloads with SR-IOV VF or PCI PF, this plugin needs to ...`

Reply:
+1

Reply:
otherwise LGTM.